shex/sio-model/undiagnosedShape.shex

```diff
@@ -0,0 +1,40 @@
+PREFIX : <http://purl.org/ejp-rd/cde/v020/shex/>
+PREFIX obo: <http://purl.obolibrary.org/obo/> 
+PREFIX sio: <http://semanticscience.org/resource/>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#> 
+
+:personShape IRI { 
+  a [ sio:SIO_000498 ] ;
+  sio:SIO_000228 @:role1Shape ;
```

Comment:
Just wonder if it is possible to generalize them into one `roleShape,` and that `roleShape` allows `sio:is-realized-in` with both `healthcareActivityProcessShape` and `geneticTestingProcessShape`. How do you think @rajaram5 ?

Reply:
Or we may need to distinguish `role1` and `role2` in a semantic way. Now both roles (https://github.com/ejp-rd-vp/CDE-semantic-model/blob/develop/images/sio-model/7_Undiagnosed.png) are a type of `obo: patient-role` and `sio:is-realized-in` processes, not much different from each other.

Reply:
@sxzhang1201 sounds logical to be. maybe we can indicate that a person can have more than one role

Reply:
@rajaram5 I understand that a person can have more than one role. What I mean here is that we cannot tell any difference between `role1Shape` and `role2Shape`, but can tell between `healthcareActivityPatientRoleShape` and `geneticTestingPatientRoleShape`.

Reply:
```role1Shape``` corresponds to ``` healthcareActivityPatientRoleShape``` or  ```geneticTestingPatientRoleShape```?

Reply:
```healthcareActivityPatientRoleShape``` seems to be more involved with ```phenotypeShape``` and ```geneticTestingProcessShape``` with ```variantShape``` but the conversation seems to imply the other way around